keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py

```diff
@@ -0,0 +1,562 @@
+class EfficientNetV2Backbone(Backbone):
+    """Instantiates the EfficientNetV2 architecture.
+
+    Args:
+        include_rescaling: bool, whether to rescale the inputs. If set
+            to `True`, inputs will be passed through a `Rescaling(1/255.0)`
+            layer.
+        width_coefficient: float, scaling coefficient for network width.
+        depth_coefficient: float, scaling coefficient for network depth.
+        default_size: integer, default input image size.
+        dropout_rate: float, dropout rate before final classifier layer.
+        drop_connect_rate: float, dropout rate at skip connections.
+        depth_divisor: integer, a unit of network width.
+        min_depth: integer, minimum number of filters.
+        activation: activation function.
+        block_args: list of dicts, parameters to construct block modules.
+        model_name: string, model name.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
+            to use as image input for the model.
+
+    Usage:
+    ```python
+    efficientnet = keras_cv.models.EfficientNetV2Backbone.from_preset(
+        "efficientnetv2-s"
+    )
+    images = tf.ones((1, 256, 256, 3))
+    outputs = efficientnet.predict(images)
+    ```
+    """
+
+    def __init__(
+        self,
+        *,
+        include_rescaling,
+        width_coefficient,
+        depth_coefficient,
+        default_size,
+        block_args,
+        dropout_rate=0.2,
+        drop_connect_rate=0.2,
+        depth_divisor=8,
+        min_depth=8,
+        activation="swish",
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        **kwargs,
+    ):
+        input_block_args = copy.deepcopy(block_args)
+
+        # Determine proper input shape
+        img_input = utils.parse_model_inputs(input_shape, input_tensor)
+
+        x = img_input
+
+        if include_rescaling:
+            x = layers.Rescaling(scale=1 / 255.0)(x)
+
+        pyramid_level_inputs = {}
+
+        # Build stem
+        stem_filters = round_filters(
+            filters=block_args[0]["input_filters"],
+            width_coefficient=width_coefficient,
+            min_depth=min_depth,
+            depth_divisor=depth_divisor,
+        )
+        x = layers.Conv2D(
+            filters=stem_filters,
+            kernel_size=3,
+            strides=2,
+            kernel_initializer=conv_kernel_initializer(),
+            padding="same",
+            use_bias=False,
+            name="stem_conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=BN_AXIS,
+            momentum=0.9,
+            name="stem_bn",
+        )(x)
+        x = layers.Activation(activation, name="stem_activation")(x)
+
+        # Build blocks
+        block_args = copy.deepcopy(block_args)
+        b = 0
+        blocks = float(sum(args["num_repeat"] for args in block_args))
+
+        pyramid_level_tracker = 1
```

Comment:
Instead of keeping this tracker, why don't we just make pyramid_level_outputs a list and then convert it to a dict at the end?

Reply:
Good idea -updated